xpti/src/xpti_proxy.cpp

```diff
@@ -113,7 +113,7 @@ class ProxyLoader {
 
   void *functionByIndex(int index) {
     if (index >= XPTI_INITIALIZE && index < XPTI_FW_API_COUNT) {
-      return m_dispatch_table[index];
+      return reinterpret_cast<void*> (m_dispatch_table[index]);
```

Comment:
This is to correct his:
This is to correct this:
xpti_proxy.cpp(116,14): error: implicit conversion between pointer-to-function and pointer-to-object is a Microsoft extension [-Werror,-Wmicrosoft-cast]
return m_dispatch_table[index];
^~~~~~~~~~~~~~~~~~~~~~~

Reply:
LGTM